README.md

```diff
@@ -66,10 +66,37 @@ Nx comes with local caching already built-in (check your `nx.json`). On CI you m
 Before any publish, releasing is mandatory if you want it to take any effect.
 
 1. Release
+
+You will need to be connected to github (GITHUB_TOKEN, GH_TOKEN) or with cli login.
+```
+sudo apt install gh
+gh auth login
+# GitHub.com
+# SSH
+# upload  ssh public key? skip
+# how ? login with a web browser
+# press enter, login.
+```
+
 Run `npm run dry-release` to see the changes that will be made are all good.
 Run `npm run release` to release every project independantly at once.
 
 If you want to release only one package specifically you can use the nx command available in each project called `project-release` and `project-dry-release`
+
+Check that the project name is inside nx.json, release.projects array.
```

Comment:
We should also add that for a project to be published, it needs the 'publishable' tag in its individual project.json file